lib/types.ts

```diff
@@ -10,13 +10,18 @@ export interface ICardData {
   level: number
   name: string
   type: string
+  cost?: string | undefined
   rarity?: string | undefined
   color?: string | undefined
   attack?: string | undefined
   defense?: string | undefined
-  effect?: string | undefined
-  specialEffect?: string | undefined
   ability_1?: string | undefined
```

Comment:
array of abilities or mapping makes more sense in my head with this quick and dirty read through

Reply:
Yup - see comments above, I think in a future PR we'll deprecate these fields.